src/test/integration/tests/Slashed_Eigenpod.t.sol

```diff
@@ -0,0 +1,267 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.27;
+
+import "src/test/integration/IntegrationChecks.t.sol";
+
+contract SlashedEigenpod is IntegrationCheckUtils {
+    using ArrayLib for *;
```

Comment:
All integration tests should start with `Integration_Name`

... that way we can run integration tests separately by doing `forge t --mc Integration`